### PR TITLE
initial setup for training maquetado

### PR DESCRIPTION
**Pull request by andyOlarte514**
# Summary

- Implementation of color variables

- Import font Roboto of Google fonts
- implementation of elements with fixed size
- add asset folder to the project
- implements `equalizer.scss`. You can see the original code here: [equalizer](https://codepen.io/SKOLZ/pen/pWJjRL)
- add necesary archives to `.gitignore`

# Trello Card

- [Setup](https://trello.com/c/Y7u0rGKV)

- [Equalizer](https://trello.com/c/FZ1Y7NkK)

# Screenshots
image

### CODE REVIEW

File: training-maquetado/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body>
+  <h1 class="title">Hello World</h1>
+</body>
+
+</html>
```
Comment (CarlosHWolox) [inline training-maquetado/index.html]: We always add a empty line at the end of the files so this icon do not appear.

File: training-maquetado/scss/application.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+@import 'equalizer';
+@import 'colors';
+@import 'styles';
```
Comment (CarlosHWolox) [inline training-maquetado/scss/application.scss]: Same here.

File: training-maquetado/scss/colors.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
```
Comment (CarlosHWolox) [inline training-maquetado/scss/colors.scss]: Same

File: training-maquetado/scss/equalizer.scss [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+li {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+::placeholder {
+  color: $equalizer-grey;
+  opacity: 1;
+}
```
Comment (CarlosHWolox) [inline training-maquetado/scss/equalizer.scss]: same

File: training-maquetado/scss/styles.scss [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  height: 24px;
+}
+
+.text-medium {
+  height: 16px;
+}
+
+.text-small {
+  height: 12px;
+}
```
Comment (CarlosHWolox) [inline training-maquetado/scss/styles.scss]: Same

File: training-maquetado/application.css [hunk 6]
```diff
@@ -0,0 +1,115 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto");
+* {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  margin: 0;
+  padding: 0; }
+
+h1,
```
Comment (CarlosHWolox) [inline training-maquetado/application.css]: this application.css should be ignored using .gitignore, only the scss folder should be available

**Review by CarlosHWolox — DISMISSED** [review]

File: training-maquetado/index.html [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
```
Comment (braiandickson) [inline training-maquetado/index.html]: Delete this empty line.

File: training-maquetado/index.html [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+<body>
+  <h1 class="title">Hello World</h1>
+</body>
+
```
Comment (braiandickson) [inline training-maquetado/index.html]: Delete this empty line.

**Review by braiandickson — CHANGES_REQUESTED** [review]

**Review by braiandickson — APPROVED** [review]

**Pull request merged**